test/cucumber/steps/FinalGradeSteps.groovy

```diff
@@ -0,0 +1,63 @@
+//package steps
+//
+//import ta.Student
+//import ta.EvaluationCriterion
+//import ta.FinalGradeController
+//
```

Comment:
Falta o 
this.metaClass.mixin(cucumber.api.groovy.Hooks)
this.metaClass.mixin(cucumber.api.groovy.EN)

sem isso não compila.